apps/condo/migrations/20240204222143-0364_remove_serviceconsumer_billingaccount_and_more.js

```diff
@@ -0,0 +1,36 @@
+// auto generated by kmigrator
+// KMIGRATOR:0364_remove_serviceconsumer_billingaccount_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMi4xIG9uIDIwMjQtMDItMDQgMTc6MjEKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMzYzX2F1dG9fMjAyNDAxMjlfMDUyOCcpLAogICAgXQoKICAgIG9wZXJhdGlvbnMgPSBbCiAgICAgICAgbWlncmF0aW9ucy5SZW1vdmVGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nc2VydmljZWNvbnN1bWVyJywKICAgICAgICAgICAgbmFtZT0nYmlsbGluZ0FjY291bnQnLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5SZW1vdmVGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nc2VydmljZWNvbnN1bWVyaGlzdG9yeXJlY29yZCcsCiAgICAgICAgICAgIG5hbWU9J2JpbGxpbmdBY2NvdW50JywKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field billingAccount from serviceconsumer
+--
+SET CONSTRAINTS "ServiceConsumer_billingAccount_71105b51_fk_BillingAccount_id" IMMEDIATE; ALTER TABLE "ServiceConsumer" DROP CONSTRAINT "ServiceConsumer_billingAccount_71105b51_fk_BillingAccount_id";
+ALTER TABLE "ServiceConsumer" DROP COLUMN "billingAccount" CASCADE;
+--
+-- Remove field billingAccount from serviceconsumerhistoryrecord
+--
+ALTER TABLE "ServiceConsumerHistoryRecord" DROP COLUMN "billingAccount" CASCADE;
```

Comment:
Please follow the migration EDR

Reply:
Hey, thanks for the reminder. I checked out migrations ADR and made this commit: 87893431590c67775d767cb867881487ae013cb7

- commented out all DROP SQL commands
- added these commands to the `fix-database-migrations-state.js` file